geps/gep-1867.md

```diff
@@ -79,7 +79,7 @@ In order to address the concerns above, I propose a standard `infrastructure` AP
 Note the important part of this is the `Gateway` change; the `GatewayClass` aspect is mostly for consistency.
 
 The exact fields are out of scope for this GEP and will be handled by additional GEPs.
-Some example GEPs already depending on this are [GEP-1713](/geps/gep-1713.md), [GEP-1651](/geps/gep-1651.md), and [GEP-1762](/geps/gep-1762.md)
+Some example GEPs already depending on this are [GEP-1713](/geps/gep-1713), [GEP-1651](/geps/gep-1651), and [GEP-1762](/geps/gep-1762)
```

Comment:
note: only 1651 exists, but 1713 and 1762 don't exist. :man_shrugging:

Reply:
Good catch, maybe just link to the first two for now?

Reply:
Actually 1713 and 1762 both exist as issues, I think they're just different branches – only 1651 landed before this. I'd vote for linking only to 1651 for now, and then we can add the others in their PRs maybe.

Reply:
Ah okay. Sure. I left the link to 1651 for now, and the other unlinked.